types/robust-point-in-polygon/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for robust-point-in-polygon 1.0
+// Project: https://github.com/mikolalysenko/robust-point-in-polygon
+// Definitions by: Crash Springfield <https://github.com/crashspringfield>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type Point = [number, number];
+
+export function robustPointInPolygon(vs: Point[], point: Point): number;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

You can then import it with

```ts
import foo = require("foo");
```

Reply:
I've updated the export to 
```
export {
  Point,
  robustPointInPolygon
};
```